ocaml/xapi/xapi_session.ml

```diff
@@ -791,25 +791,38 @@ module Caching = struct
      disabled. This function exists to delay the construction of the
      cache, as Xapi_globs configuration is not guaranteed to have been
      populated before the top-level code of this module is executed. *)
```

Comment:
Does the comment still hold for the current implementation?
I think now the configurations will always be read from the DB which will be populated when CPU hits this function.